app/backend/app.py

```diff
@@ -222,6 +222,7 @@ async def setup_clients():
     OPENAI_HOST = os.getenv("OPENAI_HOST", "azure")
     OPENAI_CHATGPT_MODEL = os.environ["AZURE_OPENAI_CHATGPT_MODEL"]
     OPENAI_EMB_MODEL = os.getenv("AZURE_OPENAI_EMB_MODEL_NAME", "text-embedding-ada-002")
+    OPENAI_EMB_DIMENSIONS = int(os.getenv("AZURE_OPENAI_EMB_DIMENSIONS", 1536))
```

Comment:
will this crash on the int() cast if the env var is a string or something?

Reply:
int('256') will not crash, but int('hi') will crash - which I think is desirable, right?

Reply:
fair